node-check/ligo-cvmfs-storage-check.sh

```diff
@@ -94,30 +82,42 @@ fi
 advertise $FS_ATTR "$RESULT" "C"
 advertise "HAS_CVMFS_IGWN_CONTAINERS" "$RESULT" "C"
 
-# Test requested by Brian
+
+info "Checking for IGWN FRAMES availability"
+
+$HAS_SINGULARITY=`get_glidein_config_value HAS_SINGULARITY`
+FS=ligo.osgstorage.org
 FS_ATTR="HAS_LIGO_FRAMES"
 RESULT="False"
-
 TEST_FILE=`shuf -n 1 client/frame_files_small.txt`
-if [[ $entry_name = "VIRGO_T2_BE_UCL_ingrid" || $entry_name = "HCC_US_Omaha_crane_gpu" ]]; then
-    setsid md5sum $TEST_FILE 1> ${FS_ATTR}.out 2> ${FS_ATTR}.err
-    if [ $? == 0 ]; then
-        if [ -s ${FS_ATTR}.out ]; then
-            cat ${FS_ATTR}.out
-            RESULT="True"
-        elif [ -s ${FS_ATTR}.err ]; then
-            cat ${FS_ATTR}.err
-        fi
-    fi
-    rm ${FS_ATTR}.out ${FS_ATTR}.err
-else
-  md5sum $TEST_FILE
-  if [ $? == 0 ]; then
+$OSG_SINGULARITY_PATH=`get_glidein_config_value OSG_SINGULARITY_PATH`
+$OSG_SINGULARITY_EXTRA_OPTS=`get_glidein_config_value OSG_SINGULARITY_EXTRA_OPTS`
+$OSG_SINGULARITY_IMAGE_DEFAULT=`get_glidein_config_value OSG_SINGULARITY_IMAGE_DEFAULT`
+TEST_CMD="head -c 1k $TEST_FILE"
+
+if [ "x$HAS_SINGULARITY" = "xTrue" ]; then
+   info "setsid $OSG_SINGULARITY_PATH exec --bind $PWD:/srv $OSG_SINGULARITY_EXTRA_OPTS $OSG_SINGULARITY_IMAGE_DEFAULT $TEST_CMD | grep Frame"
+   if ! (setsid $OSG_SINGULARITY_PATH exec --bind $PWD:/srv \
+                                           $OSG_SINGULARITY_EXTRA_OPTS \
+                                           "$OSG_SINGULARITY_IMAGE_DEFAULT" \
+                                           $TEST_CMD \
+                                           | grep Frame) 1>&2 \
+   ; then
+    RESULT="False"
+   else
     RESULT="True"
+   fi
+else
+  if ! (setsid  $TEST_CMD | grep Frame) 1>&2 \
+  ; then
+      RESULT="False"
+  else
+      RESULT="True"
   fi
 fi
```

Comment:
```suggestion
if [ "x$HAS_SINGULARITY" = "xTrue" ]; then
    info "setsid $OSG_SINGULARITY_PATH exec --bind $PWD:/srv $OSG_SINGULARITY_EXTRA_OPTS $OSG_SINGULARITY_IMAGE_DEFAULT $TEST_CMD | grep Frame"
    if ! (setsid $OSG_SINGULARITY_PATH exec --bind $PWD:/srv \
                                            $OSG_SINGULARITY_EXTRA_OPTS \
                                            "$OSG_SINGULARITY_IMAGE_DEFAULT" \
                                            $TEST_CMD \
                                            | grep Frame) 1>&2 \
    ; then
        RESULT="False"
    else
        RESULT="True"
    fi
else
    if ! (setsid  $TEST_CMD | grep Frame) 1>&2 \
    ; then
        RESULT="False"
    else
        RESULT="True"
    fi
fi
```

it would be nice to be consistent with indentation whitespace